contribution_plan/gql/gql_mutations/payment_plan_mutations.py

```diff
@@ -2,6 +2,7 @@
 from core.gql.gql_mutations.base_mutation import BaseMutation, BaseDeleteMutation, BaseReplaceMutation, \
     BaseHistoryModelCreateMutationMixin, BaseHistoryModelUpdateMutationMixin, \
     BaseHistoryModelDeleteMutationMixin, BaseHistoryModelReplaceMutationMixin
+from contribution_plan.services import PaymentPlan as PaymentPlanService
```

Comment:
Consider renaming this class in the services file. If this is imported as PaymentPlanService and not used anywhere else(to be checked) maybe renaming makes more sens.

Reply:
It was used in other files so I had to add alias to resolve conflict names. Both service and model has same name and it is common in this module that while importing it get alias